src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -63,8 +63,24 @@ private void bindListeners(ReadOnlyPerson person) {
         });
     }
 
+    /**
+     * Initializes the tags with text values and alternating colours between each index
+     * in the list
+     */
     private void initTags(ReadOnlyPerson person) {
-        person.getTags().forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+        person.getTags().forEach(tag -> {
+            Label tagLabel = new Label(tag.tagName);
+            if (Integer.parseInt(id.getText().replace(". " , "")) % 2 == 0) {
+                tagLabel.setStyle("-fx-background-color: deepskyblue;"
```

Comment:
You guys can probably make these constants, or abstract these out somehow, since you also reuse these strings in the else block